corehq/apps/userreports/reports/filters.py

```diff
@@ -39,6 +39,26 @@ def to_sql_values(self):
         }
 
 
+class NumericFilterValue(FilterValue):
+
+    def __init__(self, filter, value):
+        assert filter.type == "numeric"
+        assert (isinstance(value, dict) and "operator" in value and "operand" in value) or value is None
+        super(NumericFilterValue, self).__init__(filter, value)
+
+    def to_sql_filter(self):
+        if self.value is None:
+            return ""
+        return "{0} {1} :operand".format(self.filter.field, self.value['operator'])
```

Comment:
I check the operator a little up stream in the `NumericFilter` class on [line 167](https://github.com/dimagi/commcare-hq/pull/5407/files#diff-e4b58799e85ecb0445e2ab040416a01eR167), which I think will prevent sql injections. Might make more sense to move that check into this class though...